### PR TITLE
Refactor file descriptor handling in openFileDescriptors()

### PR DESCRIPTION
**Pull request by Java4ye**
Motivation:

The motivation behind this change is to enhance the readability and maintainability of the codebase by refactoring the file descriptor handling in the openFileDescriptors method. By extracting the logic for closing file descriptors into a separate method, closeFileDescriptor, 

Modification:

In this commit, I have refactored the openFileDescriptors method by extracting the code responsible for closing file descriptors into a separate method, closeFileDescriptor. 

Result:

This PR introduces improved code organization and readability by separating the concerns of opening and closing file descriptors in the openFileDescriptors method. It does not introduce any functional changes.

### CODE REVIEW

File: transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java [hunk 1]
```diff
@@ -137,31 +137,22 @@ public void openFileDescriptors() {
...
 
+    private static void closeFileDescriptor(FileDescriptor fd) {
+        if (fd != null) {
+            try {
+                fd.close();
+            } catch (Exception e) {
+                // ignore
+            }
+        }
+    }
```
Comment (chrisvest) [inline transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java:155]: Add a line break between this and the next method.

**Review by chrisvest — COMMENTED** [review]
Did you sign the ICLA? https://netty.io/wiki/developer-guide.html

**Comment by Java4ye** [comment]
> Did you sign the ICLA? https://netty.io/wiki/developer-guide.html

Yes, I have completed the ICLA process.

File: transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java (hunk 1 above)
Comment (Java4ye) [reply transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java:155]: ok，I made it up in the second commit

Comment (chrisvest) [reply transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java:155]: Also between this one and `newTaskQueue` below.

Comment (Java4ye) [reply transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java:155]: sorry，my carelessness

**Comment by chrisvest** [comment]
Looks like you messed up the rebase and entangled your changes with the merge of #13942. Please fix.

**Comment by Java4ye** [comment]
> Looks like you messed up the rebase and entangled your changes with the merge of #13942. Please fix.

Fixed.
Thanks.

**Review by chrisvest — APPROVED** [review]

**Comment by chrisvest** [comment]
Thanks!

**Pull request merged**